addons/mail/models/mail_message.py

```diff
@@ -1068,9 +1068,22 @@ def _extras_to_store(self, store: Store, format_reply):
         pass
 
     @api.model
-    def _message_fetch(self, domain, search_term=None, before=None, after=None, around=None, limit=30):
+    def _message_fetch(self, domain, options: dict | None = None):
```

Comment:
I don't think `_message_fetch` itself should take "options" though, this one should keep the individual params for sure.